config/pipelines.yml

```diff
@@ -67,17 +67,17 @@ default: &default
               value: uuid
             samples:
               type: :array
-              value: library
+              value: request_libraries
               children:
                 cost_code:
                   type: :model
                   value: request.cost_code
                 pac_bio_library_tube_id_lims:
                   type: :model
-                  value: id
+                  value: library.id
```

Comment:
thinking this would need to be `libraries`?

Reply:
It needs to be request libraries. Libraries on their own don't contain sufficient information such as sample name and tag information. If you look at the way libraries are created we create the library and then add all of the requests. A library can contain many requests with tags and each message needs an associated tag. Hopefully that makes sense?

Reply:
So this value should be changed to `request_libraries`? Would you loop throgh their id's?